data/common.yaml

```diff
@@ -0,0 +1,7 @@
+---
+nodejs::global_config_entries: {}
```

Comment:
please add the default value to init.pp. That enables puppet-strings to pick it up for the REFERENCE.md